components/menu/index.css

```diff
@@ -10,170 +10,676 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+
+  --spectrum-menu-item-label-icon-color-default: var(--spectrum-neutral-content-color-default);
+  --spectrum-menu-item-label-icon-color-hover: var(--spectrum-neutral-content-color-hover);
+  --spectrum-menu-item-label-icon-color-down: var(--spectrum-neutral-content-color-down);
+  --spectrum-menu-item-label-icon-color-focus: var(--spectrum-neutral-content-color-key-focus);
+
+  --spectrum-menu-item-label-content-color-disabled: var(--spectrum-disabled-content-color);
+  --spectrum-menu-item-label-icon-color-disabled: var(--spectrum-disabled-content-color);
+
+  --spectrum-menu-item-description-line-height: var(--spectrum-line-height-100);
+  --spectrum-menu-item-description-cjk-line-height: var(--spectrum-cjk-line-height-100);
+
+  --spectrum-menu-item-description-color-default: var(--spectrum-neutral-subdued-content-color-default);
+  --spectrum-menu-item-description-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
+  --spectrum-menu-item-description-color-down: var(--spectrum-neutral-subdued-content-color-down);
+  --spectrum-menu-item-description-color-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
+  --spectrum-menu-item-description-color-disabled: var(--spectrum-disabled-content-color);
+
+  --spectrum-menu-section-header-line-height: var(--spectrum-line-height-100);
+  --spectrum-menu-section-header-cjk-line-height: var(--spectrum-cjk-line-height-100);
+  --spectrum-menu-section-header-font-weight: var(--spectrum-font-weight-bold);
+  --spectrum-menu-section-header-color: var(--spectrum-gray-900);
+  --spectrum-menu-collapsible-icon-color: var(--spectrum-gray-900);
+  
+  --spectrum-menu-checkmark-icon-color-default: var(--spectrum-accent-color-900);
+  --spectrum-menu-checkmark-icon-color-hover: var(--spectrum-accent-color-1000);
+  --spectrum-menu-checkmark-icon-color-down: var(--spectrum-accent-color-1100);
+  --spectrum-menu-checkmark-icon-color-focus: var(--spectrum-accent-color-1000);
+
+  --spectrum-menu-drillin-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
+  --spectrum-menu-drillin-icon-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
+  --spectrum-menu-drillin-icon-color-down: var(--spectrum-neutral-subdued-content-color-down);
+  --spectrum-menu-drillin-icon-color-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
+
+  --spectrum-menu-item-value-color-default: var(--spectrum-neutral-subdued-content-color-default);
+  --spectrum-menu-item-value-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
+  --spectrum-menu-item-value-color-down: var(--spectrum-neutral-subdued-content-color-down);
+  --spectrum-menu-item-value-color-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
+
+  /* "one" icon: chevron + additional icon (we don't count the chevron as an icon because it HAS to be there for a collapisble) */
+  --spectrum-menu-item-collapisble-has-icon-submenu-item-padding-x-start:
```

Comment:
These comments and property names are very helpful for understanding the calc! 👏  Taking notes ✍️